### ISSUE TITLE
Why deprecate tf.layers.dropout?

### ISSUE THREAD

**Issue by chwang85** [tensorflow/tensorflow#26149]
In the tensorflow 1.13 document of tf.layers.dropout, there is a warning: 
Warning: THIS FUNCTION IS DEPRECATED. It will be removed in a future version. Instructions for updating: Use keras.layers.dropout instead.

It is ridiculous to deprecate tf.layers.dropout, because it is widely used, and very easy to use.
Do you really care about the requirements of TF developers?
Do you really care about the requirements of TF developers?
Do you really care about the requirements of TF developers?

**Comment by ymodak** [comment]
Duplicate#26151
